### PR TITLE
Custom title pr 968

### PR DESCRIPTION
**Pull request by emmalearnscode**
## Fixes Issue #964 

The creator of PR #968 never came back to fix a merge conflict although the changes were very good. Instead I created a new branch, merged their changes there and fixed the conflicting files (package-lock.json). I have also merged in all current changes since the PR was from January. Prettier has adjusted some of the files, but the main changes were to the user profile.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1475"></a>

### CODE REVIEW

File: src/Components/UserProfile/User.js [hunk 1]
```diff
@@ -19,12 +19,18 @@ function User({ singleUser }) {
   useEffect(() => {
     fetch(`/data/${username}.json`)
       .then((response) => response.json())
-      .then((data) => setProfile(data))
+      .then((data) => {
+        setProfile(data)
+        document.title = `${data.name} | Link Free`
```
Comment (schmelto) [inline src/Components/UserProfile/User.js:24]:
```suggestion
        document.title = `${data.name} | LinkFree`
```

File: src/Components/UserProfile/User.js [hunk 2]
```diff
@@ -19,12 +19,18 @@ function User({ singleUser }) {
...
+      })
       .catch(() => setError(true))
       .finally(() => {
         setShowProgress(false)
         setTimeout(() => setskeleton(false), 500)
       })
+    return () => {
+      document.title = 'Link Free'
```
Comment (schmelto) [inline src/Components/UserProfile/User.js:32]:
```suggestion
      document.title = 'LinkFree'
```

**Comment by emmalearnscode** [comment]
Well spotted Tom, updated it now.

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by eddiejaoude — APPROVED** [review]
LGTM 👍

**Pull request merged**